embedded-graphics/CHANGELOG.md

```diff
@@ -55,6 +56,7 @@ Embedded Graphics is a `no_std` library for adding graphics features to display
 - **(breaking)** [#409](https://github.com/jamwaffles/embedded-graphics/pull/409) The `DrawTarget` trait was moved from the root of the crate to a new `draw_target` module. Projects that use the `prelude` to import `DrawTarget` won't be affected by this change.
 - **(breaking)** [#420](https://github.com/jamwaffles/embedded-graphics/pull/420) To add support for additional image formats now `ImageDrawable` and `OriginDimensions` needs to be implemented instead of `IntoPixelIterator` and `ImageDimensions`.
 - **(breaking)** [#420](https://github.com/jamwaffles/embedded-graphics/pull/420) The color type parameter was removed from the `Image` struct and is now defined by the associated `Color` type of an `ImageDrawable` implementation.
+- **(breaking)** [#439](https://github.com/jamwaffles/embedded-graphics/pull/439) When text color is omitted, the text is rendered as transparent instead of a solid block using the backgorund color.
```

Comment:
IMO this PR fixes a bug so this should go under `### Fixed`, without the **(breaking)** tag. There's also a typo:

```suggestion
- **(breaking)** [#439](https://github.com/jamwaffles/embedded-graphics/pull/439) When text color is omitted, the text is rendered as transparent instead of a solid block using the background color.
```